spec/factories/work_weeks.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :work_week do
+    label                   { FFaker::Lorem.sentence }
```

Comment:
I wonder if using `FFaker` makes sense here 🤔  The values we have are two set sets that only make sense in conjunction....